client/my-sites/stats/site.jsx

```diff
@@ -144,7 +144,6 @@ class StatsSite extends Component {
 	};
 
 	onChangeLegend = ( activeLegend ) => this.setState( { activeLegend } );
-	onChangeChartQuantity = ( customChartQuantity ) => this.setState( { customChartQuantity } );
```

Comment:
This was used when the date picker was calculating the number of days between the selected dates but got replaced with URL parameters, and the calculation was moved here.